tests/cypress/e2e/unit_tests/user.spec.ts

```diff
@@ -29,6 +29,10 @@ describe('User role testing', () => {
     it('Create standard user', () => {
       cy.login();
       cypressLib.burgerMenuToggle();
-      cypressLib.createUser(stdUser, uiPassword);
+      cy.accesMenuSelection('Continuous Delivery', 'Cluster Groups');
```

Comment:
I don't understand this.
The `user.spec.ts` is not used in our current ci.
Furthermore, it has been changed its behavior honoring the standard user creation to something different. 

Why?

Reply:
it was a leftover, please don't confuse. I modified it for my changes....Sorry for creating confusion here :disappointed: